src/test/java/com/programmers/commandline/domain/consumer/repository/impl/ConsumerNamedJdbcRepositoryTest.java

```diff
@@ -0,0 +1,165 @@
+package com.programmers.commandline.domain.consumer.repository.impl;
+
+import com.programmers.commandline.domain.consumer.entity.Consumer;
+import org.junit.jupiter.api.*;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+
+@SpringBootTest
+@Transactional
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+class ConsumerNamedJdbcRepositoryTest {
+
+    @Autowired
+    private ConsumerNamedJdbcRepository consumerNamedJdbcRepository;
+
+    @BeforeEach
+    void setup() {
+        consumerNamedJdbcRepository.deleteAll();
+    }
+
+    @Test
+    @DisplayName("소비자를 추가를 검증하라")
+    public void testInsert() {
+        //given
+        Consumer consumer = new Consumer(UUID.randomUUID(), "test_user", "test_user@naver.com", LocalDateTime.now());
+
+        //when
+        consumerNamedJdbcRepository.insert(consumer);
+        Optional<Consumer> optionalConsumer = consumerNamedJdbcRepository.findById(consumer.getConsumerId());
+
+        //then
+        assertThat(optionalConsumer.isEmpty(), is(false));
+    }
+
+    @Test
+    @DisplayName("소비자를 저장하고 해당 소비자를 업데이트 합니다, 그리고 업데이트 한 이름과 이메일을 검증하라")
+    void update() {
+        //given
+        Consumer consumer = new Consumer(UUID.randomUUID(), "test_user", "test_user@naver.com", LocalDateTime.now());
+
+        //when
+        String updateName = "update_user";
+        String updateEmail = "update_user@navr.com";
+
+        consumerNamedJdbcRepository.insert(consumer);
+        consumer.update(updateName, updateEmail);
+
+        Consumer updateConsumer = consumerNamedJdbcRepository.update(consumer);
+
+        //then
+        assertThat(updateConsumer.getName(), is(updateName));
+        assertThat(updateConsumer.getEmail(), is(updateEmail));
```

Comment:
```suggestion
assertAll(
            () -> assertThat(updateConsumer.getName(), is(updateName)),
            () -> assertThat(updateConsumer.getEmail(), is(updateEmail))
        );
```
http://amazingguni.github.io/blog/2016/06/Clean-code-11-%EC%8B%9C%EC%8A%A4%ED%85%9C
해당 글에 테스트당 assert 하나 라는 말이 있긴 한데 이것 때문에 묶으라는 말은 아니고

다음을 추천하는 이유는, 테스트가 name에서 실패하면 email 검증을 pass하기 때문에 테스트 실패의 원인을 전부 파악하기 힘들 수 있습니다. (결국손이한번더가게됨) 

따라서 name과 email 두 가지가 소비자임을 검증하는 한 단위라면 묶어서 전부 테스트 해주는 것이 좋다고 생각합니다.

Reply:
감사합니다. 하나의 객체(소비자) 에서 발생하는 실패원인을 찾는 것이지 객체를 name, email로 나누어서 찾는 건 좋지 않다는 생각을 하게 되었습니다.